PlayCover/ViewModel/AppFolderVM.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  AppFolderVM.swift
+//  PlayCover
+//
+//  Created by Edoardo C. on 04/03/25.
+//
+
+class AppFolderVM: ObservableObject {
+    static let shared = AppFolderVM()
+    @Published var folderWrap = Folder(name: "", icon: "")
+    @Published var folders: [Folder] = [] {
+        didSet {
+            encode()
+        }
+    }
+
+    static let plistFolderApps = PlayTools.playCoverContainer
+        .appendingPathComponent("appFolders")
```

Comment:
Can the file name be changed to CamelCase? This is to be consistent with the naming of the `Sources` folder.